dnf/cli/commands/group.py

```diff
@@ -382,11 +382,20 @@ def run(self, extcmds):
         if cmd == 'install':
             types, patterns = self._split_extcmds(extcmds)
             self._remark = True
-            return self.base.env_group_install(patterns, types)
+            try:
+                installed = self.base.env_group_install(patterns, types)
+            except dnf.exceptions.CompsError:
+                raise dnf.exceptions.Error(_('Nothing to do.'))
+            if installed == 0:
+                raise dnf.exceptions.NothingToDo(_('Nothing to do.'))
+            return installed
```

Comment:
Hm, I don't like this change.
Why is it needed? Is it because it causes the prompt when there is no group to be installed? Did you investigate why ```Persistor.diff()``` finds some differences? Is it correct?